packages/expo-updates/android/src/androidTest/java/expo/modules/updates/launcher/DatabaseLauncherTest.kt

```diff
@@ -39,6 +40,69 @@ class DatabaseLauncherTest {
     db.close()
   }
 
+  @Test
+  fun testGetUpdateIds_EmptyDB() {
+    val launcher = DatabaseLauncher(
+      UpdatesConfiguration(null, null),
+      File("test"),
+      FileDownloader(context),
+      SelectionPolicy(
+        mockk(),
+        mockk(),
+        mockk()
+      )
+    )
+    val readyUpdateIds = launcher.getReadyUpdateIds(db)
+    Assert.assertEquals(0, readyUpdateIds.size)
+  }
+
+  @Test
+  fun testGetUpdateIds_DBWithOneUpdate() {
+    val testUpdate = UpdateEntity(UUID.randomUUID(), Date(), "1.0", "scopeKey")
+    testUpdate.lastAccessed = Date(Date().time - 24 * 60 * 60 * 1000) // yesterday
```

Comment:
```suggestion
    testUpdate.lastAccessed = Date(Date().time - DateUtils.DAY_IN_MILLIS) // yesterday
```
I think you can use a `DateUtils` class here - https://developer.android.com/reference/android/text/format/DateUtils#DAY_IN_MILLIS